eoscostarica/gGoods
# feat(smart-contracts): add base contract from dgoods

### add base contract from dgoods

### What does this PR do?

- Resolve #28 

### Steps to test

1. cd contracts/dgoods/
2. make build
3. check the folder contracts/dgoods/build

#### CheckList
